applications/FluidDynamicsApplication/custom_processes/distance_smoothing_process.h

```diff
@@ -238,7 +231,9 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) DistanceSmoothingProcess : public P
 
                 if (contact_proxy.Get(global_pointer) == rNode.Is(CONTACT)){
 
-                    const auto x_j = coordinate_proxy.Get(global_pointer);
+                    auto result = combined_proxy.Get(global_pointer);
+                    const auto x_j = std::get<1>(result);
```

Comment:
```suggestion
                    const array_1d<double,3>& x_j = std::get<1>(result);
```
We experienced some weird stuff with the auto and array types in the past. Besides, the & avoids the copy and allocation.